### PR TITLE
Issue #13213: Remove '//ok' comments from Input files `InputRequireThisTryWithResourcesOnlyOverlappingFalse`

### PR DESCRIPTION
**Pull request by abhishekmaity**
Towards #13213

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java [hunk 1]
```diff
@@ -20,22 +20,22 @@ public class InputRequireThisTryWithResourcesOnlyOverlappingFalse implements Aut
 
     void oneResource() {
         try (BufferedReader fldBufferedReader
-                     = new BufferedReader(new InputStreamReader(null, "utf-8"))) { } // ok
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java:23]: @abhishekmaity ,
It is required to remove the respective suppressions from the file [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml)

Please look at previously merged PRs for examples of update : 
- https://github.com/checkstyle/checkstyle/pull/14258
- https://github.com/checkstyle/checkstyle/pull/14259
- https://github.com/checkstyle/checkstyle/pull/14260

**Review by MANISH-K-07 — COMMENTED** [review]
Items :

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java (hunk 1 above)
Comment (abhishekmaity) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java:23]: @MANISH-K-07 Thanks, corresponding changes made at .xml file

File: config/checkstyle-input-suppressions.xml [hunk 2]
```diff
@@ -210,35 +210,7 @@
...
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisTryWithResourcesOnlyOverlappingFalse.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisTryWithResourcesOnlyOverlappingFalse.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisTryWithResourcesOnlyOverlappingFalse.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisTryWithResourcesOnlyOverlappingFalse.java"/>
-  <suppress id="UnnecessaryOkComment"
+    <suppress id="UnnecessaryOkComment"
```
Comment (abhishekmaity) [inline config/checkstyle-input-suppressions.xml:213]: The following is/are removed from `checkstyle-input-supproession.xml`
```java
<suppress id="UnnecessaryOkComment"  files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisTryWithResourcesOnlyOverlappingFalse.java"/>
```

[the remark this answers is not in the document]
Comment (abhishekmaity) [reply config/checkstyle-input-suppressions.xml:213]: I have found no changes in the indendation. Please raise a code 'request change' instead.

**Comment by nrmancuso** [comment]
@abhishekmaity I am putting this PR into draft mode while you make CI happy

**Review by nrmancuso — CHANGES_REQUESTED** [review]
@abhishekmaity this PR should have exactly one commit referencing the issue, with only `// ok` comments and associated suppressions removed.

**Comment by abhishekmaity** [comment]
> nrmancuso

@nrmancuso I have squashed 3 commits into 1 commit and re-based it. Please review the PR

**Comment by abhishekmaity** [comment]
I am fixing alll the issues

**Comment by abhishekmaity** [comment]
This PR making too much complex. closing and creating a new.

**Comment by nrmancuso** [comment]
> This PR making too much complex. closing and creating a new.

This is not necessary, please do not open and close PRs. You can fix everything via `git`.

**Pull request closed without merging**